### ISSUE TITLE
t/format.t fails for non-English locale (0.01_02-TRIAL)

### ISSUE THREAD

**Issue by eserte** [barefootcoder/date-easy#7]
This happens at least on FreeBSD systems with a German locale:
```
#   Failed test 'strftime format: %l:%M:%S%p'
#   at t/format.t line 26.
#          got: ' 4:05:06am'
#     expected: ' 4:05:06AM'
# Looks like you failed 1 test of 4.
t/format.t .............
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/4 subtests
```

Note:
```
$ env LC_ALL=C date +%p
PM
$ env LC_ALL=de_DE.UTF-8 date +%p
pm
```

**Comment by barefootcoder** [comment]
Ah, I did not realize that `%p` was locale-specific.  I'll have to avoid it in that test file.

Thx again for another great error report.

**Comment by barefootcoder** [comment]
Fixed.  Will close once CPAN Testers verifies.

**Comment by barefootcoder** [comment]
Fixed in 0.02.
